spec/presenters/result_set_presenter_spec.rb

```diff
@@ -335,6 +335,94 @@
     end
   end
 
+  describe "#top_results?" do
+    subject(:presenter_with_top_result) { ResultSetPresenter.new(finder, filter_params, view_context) }
+
+    let(:filter_params) { { keywords: 'test', order: '-relevance' } }
+    let(:results) do
```

Comment:
Do you need this? `results` is being reassigned on line 411, so I don't think this is being used.